tests/modules/cooler/zoomify/test.yml

```diff
@@ -0,0 +1,7 @@
+- name: cooler zoomify test_cooler_zoomify
+  command: nextflow run tests/modules/cooler/zoomify -entry test_cooler_zoomify -c tests/config/nextflow.config
+  tags:
+    - cooler
+    - cooler/zoomify
+  files:
+    - path: output/cooler/test.mcool
```

Comment:
Can you please add an md5sum, or, if it is not stable between runs, some keyword(s) that should be contained in the file?

Reply:
Yes it is not stable between runs. I will add some keyword(s) to validate the output.